### ISSUE TITLE
strconv.Atoi: parsing "": invalid syntax

### ISSUE THREAD

**Issue by aminvakil** [ravro-ir/ravro_dcrpt#1]
Key is present in key folder, and `ravro_dcrypt -init=init` has been executed before.
```
ls encrypt
report-ir2000-00-00-0001.zip
```
```
$ ravro_dcrpt -in=encrypt/ -out=decrypt/ -key=key/abcde.key 
>> Version : ravro_dcrpt/1.0.0
>> HomePage: https://github.com/ravro-ir/ravro_dcrp
>> Issue: https://github.com/ravro-ir/ravro_dcrp/issues
>> Author : Ramin Farajpour Cami
>> Help : ravro_dcrpt --help 


[++++] Starting for decrypting Report . . . 
[++++] Starting for decrypting Judgment . . . 
[++++] Starting for decrypting Amendment . . . 
[++++] decrypted successfully 
2022/07/17 10:57:23 strconv.Atoi: parsing "": invalid syntax
```
Package has been made with this command:
```
go build ravro_dcrpt
```
Also a couple of versions if necessary:
```
go version go1.18.3 linux/amd64
OpenSSL 1.1.1q  5 Jul 2022
wkhtmltopdf 0.12.6
```

**Comment by raminfp** [comment]
Hi @aminvakil ,

Thanks for report, I fixed https://github.com/ravro-ir/ravro_dcrp/commit/426c652a0622d4716814e76a1d9ed24c4e1acb14 this problem today.  Do you have the latest version?

and please define report directory, for example :
`ravro_dcrp -in=encrypt/report-ir2022-07-16-0002 -out=decrypt/ -key=key/key.private` you use `ravro_dcrpt -in=encrypt/ -out=decrypt/ -key=key/abcde.key ` which is incorrect.

**Comment by aminvakil** [comment]
> Hi @aminvakil ,
> 
> Thanks for report, I fixed [426c652](https://github.com/ravro-ir/ravro_dcrp/commit/426c652a0622d4716814e76a1d9ed24c4e1acb14) this problem today. Do you have the latest version?

Yes.
> 
> and please define report directory, for example : `ravro_dcrp -in=encrypt/report-ir2022-07-16-0002 -out=decrypt/ -key=key/key.private` you use `ravro_dcrpt -in=encrypt/ -out=decrypt/ -key=key/abcde.key ` which is incorrect.

Extracting the report and defining directory worked, thanks!
